lib/reek/spec/should_reek_of.rb

```diff
@@ -9,20 +9,30 @@ module Spec
     # code smell.
     #
     class ShouldReekOf
+      # Variant of Examiner that doesn't swallow exceptions
+      class UnsafeExaminer < Examiner
+        def run
+          examine source.syntax_tree
+          smell_repository.report_on(collector)
```

Comment:
This will break with #1041 merged - we probably should get #1041 merged first and then update this one.